addons/laser/CfgVehicles.hpp

```diff
@@ -10,4 +10,19 @@ class CfgVehicles {
             };
         };
     };
+
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class ACE_ToggleMarkerLaser {
+                    displayName = CSTRING(laserMarkToggle);
+                    condition = QUOTE('Laserdesignator' in weapons ACE_player);
```

Comment:
Probably want something like `getNumber (configFile >> "CfgWeapons" >> (typeOf binocular ACE_PLAYER) >> QGVAR(whateverconfigname)) == 1` so that mods can add functionality without a headache